.teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt

```diff
@@ -30,7 +32,7 @@ class CIConfigIntegrationTests {
     fun macOSBuildsSubset() {
         val m = CIBuildModel()
         val p = RootProject(m)
-        val releaseAccept = p.subProjects.find { it.name.contains("Release Accept") }!!
+        val releaseAccept = p.subProjects.find { it.name.contains("Ready for Release") }!!
```

Comment:
Why not `READY_FOR_RELEASE` here?

Reply:
Better now?

Reply:
Yeah, I realized that. That is the current code in this PR (I force pushed):
```
 val readyForRelease = p.subProjects.find { it.name.contains(StageNames.READY_FOR_RELEASE.stageName) }!!
```

Is that good enough? Or did you expect something else?